### PR TITLE
Hh layout-worked on all layout elements

### PR DESCRIPTION
**Pull request by hhassan1230**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,33 +3,39 @@
...
+            <nav>
+              <ul>
+                <li><a href="#" title="OVERVIEW">OVERVIEW</a></li>
+                <li><a href="#" title="CURRICULUM">CURRICULUM</a></li>
+                <li><a href="#" title="APPLICATION PROCESS">APPLICATION PROCESS</a></li>
+                <li><a href="#" title="KEY DATES">KEY DATES</a></li>
+              </ul>
+            </nav>  
+          </div>
+              <!-- <img src="http://placehold.it/1410x350"> -->
```
Comment (sarony) [inline index.html]: Let's remove the comment.

File: index.html [hunk 2]
```diff
@@ -69,56 +77,60 @@
...
-          <img src="http://placehold.it/140x100"><h3>Curriculum Info 4</h3><p>Info blub</p>
-        </div>
-        <div class="image-blurb">
-          <img src="http://placehold.it/140x100"><h3>Curriculum Info 5</h3><p>Info blub</p>
-        </div>
-        <div class="image-blurb">
-          <img src="http://placehold.it/140x100"><h3>Curriculum Info 6</h3><p>Info blub</p>  
+        <div class="curriculum-blurbs">
+          <div class="image-blurb">
+            <img src="http://placehold.it/140x100"><h3>Curriculum Info 1</h3><p>Info blub</p>
```
Comment (sarony) [inline index.html]: Let's add line breaks between html elements.

File: index.html [hunk 3]
```diff
@@ -69,56 +77,60 @@
...
+                <p>Do you meet the following criteria</p>
+              </li>
+              <li>
+                <h3>Fill out application form</h3>
+                <p>The application helps us get to know you better. Answer questions about your interest in technology and what gets you excited about the program.</p>
+              </li>
+              <li>
+                <h3>Take entrace assessment</h3>
+                <p>Learn the fundamentals of networking and take this online entrace assessment to prove your knowledge.</p>
+              </li>
```
Comment (sarony) [inline index.html:122]: Let's add an application step:

Interview
Selected candidates will be invited to interview for a spot in the cohort.

**Comment by sarony** [comment]
Let's make those minor changes, then feel free to merge.

File: index.html (hunk 3 above)
Comment (hhassan1230) [inline index.html:122]: added step

**Pull request merged**